app/src/androidTest/java/com/sdp/movemeet/MainActivityTest.java

```diff
@@ -102,6 +122,20 @@ public void mainActivityToStartActivity() {
     @Test
     public void mainActivityGotoHome() {
         onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.nav_start_activity));
+
+        /*Context context = ApplicationProvider.getApplicationContext();
+
+        Intent activity = new Intent(context, ActivityDescriptionActivity.class);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_ACTIVITY_ID ,DUMMY_ACTIVITY_ID);
+        activity.putExtra(String.valueOf(ActivityDescriptionActivity.EXTRA_LATITUDE),DUMMY_LATITUDE);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_DATE ,DUMMY_DATE);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_TITLE ,DUMMY_TITLE);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_DESCRIPTION ,DUMMY_ACTIVITY_ID);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_ACTIVITY_ID ,DUMMY_ACTIVITY_ID);
+        activity.putExtra(ActivityDescriptionActivity.EXTRA_ACTIVITY_ID ,DUMMY_ACTIVITY_ID);
+
+        ActivityScenario.launch(activity);*/
+
```

Comment:
why all this commented bloc ? is it still useful to keep it ?

Reply:
Yes, but I will finish this task next week